services/notebooks/notebooks_service.py

```diff
@@ -49,7 +49,9 @@
 def _server_name(namespace, project, commit_sha):
     """Form a DNS-safe server name."""
     escape = lambda x: escapism.escape(
-        x, safe=set(string.ascii_lowercase + string.digits)
+        x,
+        safe=set(string.ascii_lowercase + string.digits),
+        escape_char='-',
```

Comment:
https://docs.python.org/3/library/functools.html#functools.partial

```python
escape = partial(escapism.escape, safe=..., escape_char=...)
```

Reply:
no, I was lazy for not doing it that way ;)